colossalai/fx/profiler/experimental/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .registry import meta_profiler_function, meta_profiler_module
-from .memory import calculate_fwd_in, calculate_fwd_tmp, calculate_fwd_out
+from .memory import calculate_fwd_in, calculate_fwd_out, calculate_fwd_tmp
+from .profiler import profile_function, profile_method, profile_module
 from .profiler_function import *
 from .profiler_module import *
-from .profiler import profile_function, profile_method, profile_module
+from .registry import meta_profiler_function, meta_profiler_module
```

Comment:
Add some documentation in this file to explain why this folder is `experimental`.

Reply:
I might refactor this `experimental/` folder later, it hasn't been updated for a long time.